src/Logging/Logging.Console/Extensions/LogServiceFactory.Console.Extensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using CodeMonkeys.Logging.Console;
+
+namespace CodeMonkeys.Logging
+{
+    public static partial class LogServiceFactoryExtensions
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5626772)